addon/activity-notifier/service.js

```diff
@@ -8,15 +8,25 @@ import Configuration from '@upfluence/ember-upf-utils/configuration';
 const notificationMessage = function (message) {
   return {
     title: `<i class="toast-title__icon upf-icon upf-icon--messages"></i>`,
-    message: message
+    message: message,
+    type: 'info'
+  };
+};
+
+const notificationErrorMessage = function (message) {
```

Comment:
```suggestion
function notificationErrorMessage(message) {
```

Out of curiosity, why the `const` based function definition ? :thinking:

Reply:
ideally with const it would be an arrow function. probably just old code that hasn't moved to new styles. It would be:

```
const notificationErrorMessage = (message) => {
  .... 
}

Reply:
To be honest, no idea, I used the same as the other functions in the file 
Maybe @phndiaye could lighten us here ?